src/Views/DriversView.vala

```diff
@@ -0,0 +1,114 @@
+/*-
+ * Copyright 2023 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ public class DriversView : AbstractInstallerView {
+    public signal void next_step ();
+
+    construct {
+        var image = new Gtk.Image.from_icon_name ("application-x-firmware", Gtk.IconSize.INVALID) {
+            pixel_size = 128,
+            valign = Gtk.Align.END
+        };
+
+        var title_label = new Gtk.Label (_("Install Drivers")) {
```

Comment:
Just to make it clear that we're not installing drivers immediately in this step, but making a choice about installing them later. Going with the language "additional" here to imply that the system already comes with drivers, but there are more available optionally

```suggestion
        var title_label = new Gtk.Label (_("Additional Drivers")) {
```